runbook/runbook.sh

```diff
@@ -0,0 +1,298 @@
+#!/bin/bash
+
+set -uo pipefail
+
+SCRIPT="$0"
+NARGS="$#"
+
+if [ "$NARGS" -ne 0 ]; then
+  echo "error: extra arguments"
+  echo
+  usage
+  exit 1
+fi
+
+
+if [ -z "${DCOS_VERSION+x}" ]; then
+  exec /opt/mesosphere/bin/dcos-shell "$SCRIPT"
+fi
+
+usage() {
+  echo "usage: $SCRIPT"
+  echo
+  echo "a networking helper for troubleshooting and collecting diagnostics data"
+  echo
+  echo "available commands:"
```

Comment:
No subcommands are available (just data output).

Reply:
Thanks for catching this!